include/dmlc/base.h

```diff
@@ -37,6 +37,14 @@
 #define DMLC_LOG_CUSTOMIZE 0
 #endif
 
+/*!
+ * \brief Wheter to print stack trace for fatal error,
+ * enabled on linux when using gcc.
+ */
+#if (!defined(DMLC_LOG_STACK_TRACE) && defined(__GNUC__))
```

Comment:
A check for `&& !defined(__MINGW32__)` or `&& !defined(WIN32)` might be needed. I assume that 64 bit versions have those defined too.